src/pages/About.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
```

Comment:
I think you might have to edit the karma config file to compile `src/**/*.tsx` instead of just `src/components/**/*.tsx` for this to work with the test suite.

Reply:
Thanks fixed it! I was searching in the gulpfile :(